demo/index.md

```diff
@@ -3,34 +3,30 @@ layout: default
 markdown: kramdown
 permalink: demo/
 ---
-# Angry Bots Demo
-<div markdown="1" id="wasm-fail" class="flash flash-error flash-hide">
-  Uh-oh! WebAssembly isn't enabled in this browser. To get an early preview of this experimental technology, at your own risk:
```

Comment:
see screenshot in PR description.

Here's what it looks like when wasm is enabled:

![screen shot 2017-03-06 at 11 37 39 am](https://cloud.githubusercontent.com/assets/970121/23626343/61cb09be-0261-11e7-84b3-ec2a3e1051e4.png)